apps/emqx/src/emqx_schema.erl

```diff
@@ -1966,7 +1966,7 @@ mqtt_parse_options() ->
             sc(
                 hoconsc:enum([chunk, frame]),
                 #{
-                    default => <<"chunk">>,
+                    default => <<"frame">>,
```

Comment:
I think also those testcases might need to be changed:
* `emqx_listeners_SUITE:t_tcp_frame_parsing_conn/1` (to `t_tcp_chunk_parsing_conn/1` perhaps?)
* `emqx_listeners_SUITE:t_ssl_frame_parsing_conn/1`
* `emqx_listeners_update_SUITE:t_tcp_change_parse_unit/1`
* `emqx_listeners_update_SUITE:t_ssl_change_parse_unit/1`